.gitignore

```diff
@@ -75,3 +75,20 @@ bower_components/
 .grunt/
 src/vendor/
 dist/
+
+# Model generated OUT & RUN files #
+###################################
+refs/original_code_c/hyd.out
+refs/original_code_c/topmod.out
+hyd.out
+topmod.out
+run_bmi
+z_trash
+test/hyd.out
+test/topmod.out
+test/run_bmi_test
+test/z_trash_test
+
+# Misc #
+########
+make_and_run_bmi.sh
```

Comment:
I suspect the file _make_and_run_bmi.sh_ needs to be removed from the repo, based on _INSTALL.md_.  That or this line in _.gitignore_ should probably be removed. 

 If it was intended to have the file in the repo _and_ have it be ignored, I think an explicit comment should be added to explain things.

Reply:
Now using Make to build/run so I'll just remove shell script completely. Good catch.